src/v0/destinations/iterable/data/IterablePageConfig.json

```diff
@@ -28,7 +28,8 @@
   },
   {
     "destKey": "createdAt",
-    "sourceKeys": "originalTimestamp",
+    "sourceKeys": "createdAt",
```

Comment:
do we get `createdAt` all the time or should we have fallback to `timestamp` ?

Reply:
createdAt is optional. We are mapping this from generic mapping

`["traits.createdAt", "context.traits.createdAt", "timestamp", "originalTimestamp"]`